go.sum

```diff
@@ -771,6 +785,12 @@ github.com/onsi/ginkgo v1.10.3/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+
 github.com/onsi/ginkgo v1.11.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+WWjE=
 github.com/onsi/ginkgo v1.12.1/go.mod h1:zj2OWP4+oCPe1qIXoGWkgMRwljMUYCdkwsT2108oapk=
 github.com/onsi/ginkgo v1.14.0/go.mod h1:iSB4RoI2tjJc9BBv4NKIKWKya62Rps+oPG/Lv9klQyY=
+<<<<<<< HEAD
```

Comment:
You need to resolve all conflicts here, use an editor for Git maybe that’s easier

Reply:
Running `go mod tidy` should do it.

Reply:
If only there was an eye-roll emoji on GitHub...

I actually DID EXACTLY THIS while merging the fix @makkes gave me for the CRD linebreaks but like an idiot I cherry-pick-merged them into the same commit.  Which of course, I then dropped later when I realized I had to undo the line-break changes, and forgot about before pushing again.

The comedy (of errors) never stops!  I'll be here all week.  Tip your waitress.